model/utils.go

```diff
@@ -304,7 +304,7 @@ func Etag(parts ...interface{}) string {
 	return etag
 }
 
-var validHashtag = regexp.MustCompile(`^(#[A-Za-zäöüÄÖÜß]+[A-Za-z0-9äöüÄÖÜß_\-]*[A-Za-z0-9äöüÄÖÜß])$`)
+var validHashtag = regexp.MustCompile(`^(#\\pL[\\pL\\d\\-_.]*[\\pL\\d])$`)
```

Comment:
Your unit tests are failing. I think it's because of the extra backslashes causing the regex to be incorrect. Since they're surrounded by backquotes (like \`), you don't need to escape them. This should just be
```
var validHashtag = regexp.MustCompile(`^(#\pL[\pL\d\-_.]*[\pL\d])$`)
```